ott/core/neuraldual.py

```diff
@@ -14,17 +14,20 @@
 """A Jax implementation of the ICNN based Kantorovich dual."""
 
 import warnings
-from typing import Iterator, Optional
+from typing import Dict, Iterator, List, Optional, Tuple, Union
 
 import flax.linen as nn
 import jax
 import jax.numpy as jnp
 import optax
 from flax.core import freeze
-from flax.training import train_state
 from optax._src import base
+from typing_extensions import Literal
 
-from ott.core import icnn
+from ott.core import icnn, potentials
+
+Train_t = Dict[Literal["training_logs", "validation_logs"], List[float]]
+Potentials_t = potentials.DualPotentials
```

Comment:
what does the `_t` mean in this context?

Reply:
Just a convention to signify it's a type.